src/Mono.Android/Android.Content.PM/PackageManager.cs

```diff
@@ -0,0 +1,79 @@
+namespace Android.Content.PM;
+
+#if ANDROID_34
+public abstract partial class PackageManager
+{
+	public sealed partial class PackageInfoFlags
+	{
+		// Create overloads that accept PackageInfoLongFlags
+		[global::System.Runtime.Versioning.SupportedOSPlatformAttribute ("android33.0")]
+		public static unsafe Android.Content.PM.PackageManager.PackageInfoFlags Of (PackageInfoLongFlags value)
+			=> Of ((long) value);
+
+		[global::System.Runtime.Versioning.SupportedOSPlatformAttribute ("android33.0")]
+		public unsafe PackageInfoLongFlags ValueAsFlags
+			=> (PackageInfoLongFlags) Value;
+	}
+}
+
+// Manually created "long" version of "PackageInfoFlags" enum, created from documentation:
+// https://developer.android.com/reference/android/content/pm/PackageManager.PackageInfoFlags#of(long)
+[System.Flags]
+public enum PackageInfoLongFlags : long
```

Comment:
Should this instead be named `PackageInfoFlagsLong`, as IntelliSense code completion for tokens starting with `PackageInfoFlags` would then show `PackageInfoFlagsLong` as possible completions?

Reply:
If you've already typed out `PackageInfoFlags` then you aren't really using Intellisense!  😁 

Generally you don't have to type any part of an enum type name because Intellisense is already showing it to you because it knows the parameter type.

Regardless, either seems fine, changed.